plugins/Kaleidoscope-LED-ActiveLayerKeys/src/kaleidoscope/plugin/LED-ActiveLayerKeys.cpp

```diff
@@ -37,6 +38,10 @@ LEDActiveLayerKeysEffect::TransientLEDMode::TransientLEDMode(
   : parent_(parent),
     active_color_{0, 0, 0} {}
 
+EventHandlerResult LEDActiveLayerKeysEffect::onLedEffectQuery() {
+  return ::Focus.sendName(F("LEDActiveLayerKeysEffect"));
```

Comment:
I'm abusing `::Focus.sendName` as it's what `onNameQuery` uses and it gets the job done. I should probably rename it to `::Focus.sendString` or something similar.

Any preference?

Reply:
maybe sendLine()?  since what's special about it is that it sends a linefeed afterward?

Reply:
Makes sense. Do you want that in a separate PR?

Reply:
@obra in case you missed this: should this change be done in a separate PR? It might be breaking for custom plugins

Reply:
I think it probably makes sense to stick a symbol alias in place so it doesn't break on people?